test/test_files/tinysnb/asp/asp.test

```diff
@@ -0,0 +1,27 @@
+-NAME AspBasic
+-QUERY MATCH (a:person)-[e1:knows]->(b:person) WHERE a.age > 35 RETURN b.fName
+-ENCODED_JOIN HJ(b._id){E(b)S(a)}{S(b)}
+---- 3
+Alice
+Bob
+Dan
+
+-NAME AspMultiKey
+-QUERY MATCH (a:person)-[e1:knows]->(b:person)-[e2:knows]->(c:person), (a)-[e3:knows]->(c) WHERE a.fName='Alice' RETURN b.fName, c.fName
+#-ENCODED_JOIN HJ(c._id,b._id){E(b)E(c)S(a)}{HJ(b._id){S(b)}{E(b)S(c)}}
```

Comment:
Why comment out the ENCODED_JOIN? Remove it if not needed?

Reply:
Kept because it's the join order that will trigger ASP multi key. The same order may not be picked on every platform. But it's better to keep as a reference.